packagers/yarn-workspaces/yarn.lock

```diff
@@ -27,10 +27,10 @@
     "@jridgewell/resolve-uri" "^3.0.3"
     "@jridgewell/sourcemap-codec" "^1.4.10"
 
-"@prisma/client@4.13.0-dev.32":
```

Comment:
Unrelated: the update script has trouble with yarn, so versions are bumped but not always the lockfiles. Likely to be network concurrency issues or mismatched yarn version (not latest classic).